File: 백준/Silver/20291. 파일 정리/파일 정리.py
from sys import *
from collections import * 
from heapq import * 
from functools import *
from math import *
from itertools import *
import string

#setrecursionlimit(10**9)
input = stdin.readline

n = int(input())

_hash = defaultdict(int)

for _ in range(n):
    file,extension = input()[:-1].split(".")
    _hash[extension] += 1
    
    


for key in sorted(_hash.keys()):
    print(key,_hash[key])


    
    

File: 백준/Silver/20291. 파일 정리/파일 정리.swift
let n = Int(readLine()!)!

var files: [String] = []

var hash: [String:Int] = [:]

for _ in 0..<n {
    var s = readLine()!.split{$0 == "."}.map{String($0)}
    
    let ext = s[1]
    
    hash[ext,default: 0] += 1
    
}

for (k,v) in hash.sorted(by: {$0.key < $1.key}) {
    print("\(k) \(v)")
}
